code/game/objects/items/supermatter_delaminator.dm

```diff
@@ -0,0 +1,213 @@
+/obj/item/paper/guides/antag/antinoblium_guide
+	info = "Antinoblium Shard User's Manual<br>\
+	<ul>\
+	<li>Approach an active supermatter crystal with radiation shielded personal protective equipment. DO NOT MAKE PHYSICAL CONTACT.</li>\
+	<li>Magnetically attach the data corruptor (also provided) into the supermatter control infrastructure to allow attaching the antinoblium.</li>\
+	<li>Open the antinoblium container (provided).</li>\
+	<li>Use antinoblium extraction tongs (also provided) and apply the shard to the crystal. Take note that an EMP pulse will be emitted upon attachment. Prepare accordingly. </li>\
+	<li>Physical contact of any object with the antinoblium shard will fracture the shard and cause a spontaneous energy release.</li>\
+	<li>Extricate yourself immediately. You have approximately 5 minutes before the infrastructure fails completely.</li>\
+	<li>Upon complete infrastructure failure, the crystal well will destabilize and emit electromagnetic waves that span the entire station.</li>\
+	<li>Nanotresen safety controls will announce the destabilization of the crystal. Your identity will likely be compromised, but nothing can be done about the crystal.</li>\
+	</ul>"
+
+/obj/item/supermatter_delaminator/Initialize()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/supermatter_delaminator/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/supermatter_delaminator/antinoblium_shard
+	icon = 'icons/obj/supermatter_delaminator.dmi'
+	name = "antinoblium shard"
+	desc = "A small syndicate-engineered shard derived from supermatter. Highly fragile, do not handle without protection!"
+	icon_state = "antinoblium_shard"
+
+/obj/item/supermatter_delaminator/antinoblium_shard/attack_tk() // no TK dusting memes
+	return FALSE
+
+/obj/item/supermatter_delaminator/antinoblium_shard/can_be_pulled(user) // no drag memes
+	return FALSE
+
+/obj/item/supermatter_delaminator/antinoblium_shard/attackby(obj/item/W, mob/living/user, params)
+	if(istype(W, /obj/item/hemostat/antinoblium))
+		var/obj/item/hemostat/antinoblium/tongs = W
+		if (tongs.shard)
+			to_chat(user, "<span class='notice'>\The [tongs] is already holding an antinoblium shard!</span>")
+			return FALSE
+		forceMove(tongs)
+		tongs.shard = src
+		tongs.update_icon()
+		to_chat(user, "<span class='notice'>You carefully pick up [src] with [tongs].</span>")
+	else if(istype(W, /obj/item/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter/)) // we don't want it to dust
```

Comment:
```suggestion
	else if(istype(W, /obj/item/scalpel/supermatter) || istype(W, /obj/item/nuke_core_container/supermatter)) // we don't want it to dust
```

Reply:
Why the scalpel?

Reply:
And why the container as a matter of fact

Reply:
Copy paste code bad, fixed now.